WordPress/gutenberg
# Style Book: Close list view when opening the style book.

## What?

Closing the list block when opening the style book.

Fixes: https://github.com/WordPress/gutenberg/issues/50409

## Why?

List view is not accessible when Style Book is opened (which is good, as it's irrelevant to Style Book). 

## How?


## Testing Instructions

1. Open the site editor.
2. Edit styles.
3. Open list view when editing styles.
4. Launch style book.
5. Confirm the list view is closed when the style book is opened.

## Screenshots or screencast 

https://user-images.githubusercontent.com/252415/236861375-d569d187-509a-40f9-a06c-33423909740a.mov
